include/CL/opencl.hpp

```diff
@@ -9137,6 +9231,66 @@ typedef CL_API_ENTRY cl_int (CL_API_CALL *PFN_clEnqueueReleaseD3D10ObjectsKHR)(
         return detail::errHandler(::clFinish(object_), __FINISH_ERR);
     }
 
+#if defined(cl_khr_external_memory) && CL_HPP_TARGET_OPENCL_VERSION >= 300
+    cl_int EnqueueAcquireExternalMemObjects(
+        const vector<Memory>& mem_objects = {},
```

Comment:
Do we really want a default value for this argument?  We should properly handle the case where the list of memory objects is empty (and I think we do, below), but this won't be a common case either.  Same goes for `enqueueReleaseExternalMemObjects`.

Reply:
Right, it's not expected. Change applied.